lookout/style/format/__main__.py

```diff
@@ -48,12 +48,13 @@ def main():
     args = parser.parse_args()
     try:
         handler = args.handler
+        delattr(args, "handler")
     except AttributeError:
         def print_usage(_):
             parser.print_usage()
 
         handler = print_usage
-    return handler(args)
+    return handler(**vars(args))
```

Comment:
I don't see where `vars` comes from...

Reply:
it's to transform the `argsparse.Namespace` into a dict

Reply:
cool!